src/main/resources/application.yml

```diff
@@ -132,4 +132,5 @@ EF_SAK_SCOPE: api://${DEPLOY_ENV}-gcp.teamfamilie.familie-ef-sak/.default
 FAMILIE_EF_VEDTAK_TOPIC: teamfamilie.aapen-ensligforsorger-iverksatt-vedtak
 FAMILIE_KS_VEDTAK_TOPIC: teamfamilie.aapen-kontantstotte-vedtak-v1
 
-INNTEKTSKONTROLL_CRON_EXPRESSION: 0 0 4 6 * * # kl 04:00 den 6. hver måned
\ No newline at end of file
+INNTEKTSKONTROLL_CRON_EXPRESSION: 0 0 4 6 * * # kl 04:00 den 6. hver måned
+OPPGAVERDØDSFALL_CRON_EXPRESSION: 0 5 * * 1 * # kl 05:00 på mandager
```

Comment:
Skal vi ikke sjekke hver dag? 
Mulig jeg må lese mer kode og at dette er ok, så bare for meg at vi sjekker hver dag...

Reply:
Ok, endrer til at vi sjekker hver dag